src/collect-connect.rst

```diff
@@ -3,26 +3,22 @@ Connecting to a Server
 
 ODK Collect is used to fill forms with participants. Filled forms then need to be sent to a central location for review and analysis. Generally, organizations do this by configuring Collect to send forms to a server. For those working in environments without any internet connectivity, there are :ref:`other options <other-collect-server-options>`.
```

Comment:
Drop the last two sentences (Generally...For those working).

Reply:
Yeah, like my comment above, these feels important to maintain direction for folks needing to do offline set up.